meta-leda-components/recipes-sdv/sdv-wifi/iwd-config_git.bb

```diff
@@ -12,16 +12,22 @@
 # ********************************************************************************/
 
 DESCRIPTION = "DHCP Client configuration for WiFi with iwd"
-LICENSE = "Apache-2.0"
 
 PROVIDES:${PN} += "iwd-config"
 RPROVIDES:${PN} += "iwd-config"
 
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
+SRC_URI:append = " file://main.conf"
+
 do_install() {
     install -d ${D}/etc/iwd
-    install -m 0644 ${THISDIR}/files/main.conf ${D}/etc/iwd
+    install -m 0644 ${WORKDIR}/main.conf ${D}/etc/iwd
 }
 
+SRC_URI += "file://LICENSE"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://${WORKDIR}/LICENSE;md5=3b83ef96387f14655fc854ddc3c6bd57"
+
 PACKAGES = "${PN}"
 FILES:${PN} += "/etc/iwd/main.conf"
```

Comment:
I think this should be using `${SYSCONFDIR}`
Same for the install command in L23-24